src/securejoin.rs

```diff
@@ -631,32 +589,6 @@ pub(crate) async fn observe_securejoin_on_other_device(
                 }
                 peerstate.prefer_encrypt = EncryptPreference::Mutual;
                 peerstate.save_to_db(&context.sql).await.unwrap_or_default();
-            } else if let Some(fingerprint) =
-                mime_message.get_header(HeaderDef::SecureJoinFingerprint)
-            {
-                // FIXME: Old versions of DC send this header instead of gossips. Remove this
-                // eventually.
```

Comment:
Hum

- This is pretty quick to remove the support for this isn't it?  I don't know if we have a policy but IIRC users start to be nagged to upgrade after about 1 year, so I guess you need to wait for at least 1 year?
- It was kind of nice that you could read the securejoin protocol and get the fingerprint without having to understand the autocrypt gossip stuff.  This makes the dependence stronger, and thus increases the number of things you need to understand to work on it.  On the other hand it duplicates information and that's also confusing.  I realise I may have said something that led to this... I don't always say smart things... of course this could also be one of those.

Reply:
- Agree, let's wait for 1 year, made it a draft.
- Maybe it would be nice for the Securejoin to not depend on Autocrypt things, but anyway knowing a joiner's fingerpint isn't sufficient for other devices on a joining side to encrypt to the joiner, a key (gossip) is needed. There's also the corresponding PR to the Securejoin documentation: https://github.com/nextleap-project/countermitm/pull/85 . So, i think better not to complicate the Securejoin protocol with passing keys and rely on the Autocrypt. CC @hpk42